client/components/deposits-overview/test/index.tsx

```diff
@@ -217,18 +219,8 @@ describe( 'Deposits Overview information', () => {
 		expect( getByText( 'October 1, 2021' ) ).toBeTruthy();
 	} );
 
-	test( 'Confirm next deposit default status and date', () => {
-		const overview = createMockOverview( 'usd', 100, 0, 'rubbish' );
-
-		const { getByText } = render(
-			<NextDepositDetails isLoading={ false } overview={ overview } />
-		);
-		expect( getByText( 'Unknown' ) ).toBeTruthy();
-		expect( getByText( '—' ) ).toBeTruthy();
-	} );
-
 	test( 'Renders capital loan notice if deposit includes financing payout', () => {
-		const overview = createMockOverview( 'usd', 100, 0, 'rubbish' );
```

Comment:
This test and the one below it had "rubbish" as the deposit status, however the status had no bearing on the test. To keep this tests and appease the newly added strict type, I've updated it to use the 'estimated' deposit status instead.